README.md

```diff
@@ -29,13 +29,40 @@ and lets you
 
 ## Installation
 
+### Full Install
 One command does it all:
 
 ```sh
 bash <(curl -sL https://git.io/haskell-vim-now)
 ```
 **WARNING**: this command, once successful, will _make backups and **remove**_ your existing VIM configurations (`.vim`, plugins etc). You can later [customize](#customizing) HVN configurations.
 
+### Basic Install
```

Comment:
How about putting this at the end of the readme with a name like "Other Install Methods" or "Installing without Haskell"? It's good to have the info, but I'd like to keep the first paragraphs simple.